tests/support/identity.rs

```diff
@@ -84,10 +98,17 @@ impl Identity {
 
         env.put(app::config::ENV_IDENTITY_DIR, id_dir);
         env.put(app::config::ENV_IDENTITY_TOKEN_FILE, token);
-        env.put(app::config::ENV_IDENTITY_TRUST_ANCHORS, trust_anchors);
+        env.put(
+            app::config::ENV_IDENTITY_TRUST_ANCHORS,
+            trust_anchors.clone(),
```

Comment:
TIOLI: since `Identity::from_pem` takes an `&str`, you could call that function with `&trust_anchors` prior to inserting it into the test env, obviating the need for the clone. Since this is not called in a hot path, performance here isn't really important, but I thought I'd mention it.

Reply:
Nice ! Thanks :)